Vurfan/Encapsulation.js

```diff
@@ -0,0 +1,25 @@
+class Student {
+  constructor(name, marks) {
+    this.name = name;
```

Comment:
where is the declaration for field name and marks?

Reply:
Showing error when field name and marks are declared inside class